crypto/fipsmodule/modes/gcm_test.cc

```diff
@@ -189,6 +189,40 @@ TEST(GCMTest, ABI) {
         }
       }
     }
+    if (crypto_gcm_avx512_enabled()) {
+      CHECK_ABI_SEH(gcm_init_avx512, Htable, kH);
+      CHECK_ABI_SEH(gcm_gmult_avx512, X, Htable);
+      for (size_t blocks : kBlockCounts) {
+        CHECK_ABI_SEH(gcm_ghash_avx512, X, Htable, buf, 16 * blocks);
+      }
+
+      if (hwaes_capable()) {
+        AES_KEY aes_key;
+        static const uint8_t kKey[16] = {0};
+
+        // aes_gcm_*_avx512 makes assumptions about |GCM128_CONTEXT|'s layout.
```

Comment:
can we add a unit test to make sure the assumption is correct?

Reply:
This is inherited from the old code (line 168), not sure how can we add  UT for structure layout.